azure/requests.yaml

```diff
@@ -14,9 +14,6 @@ releases:
         - name: containerlinux
           version: ">= 2512.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13150
-        - name: external-dns
```

Comment:
Not sure how this works so bear with me. Why is this removed?

Reply:
It was added in the wrong section, for clusters < 13.0.0.
We're not going to release any new release < 13.0.0